lightning/src/ln/channelmonitor.rs

```diff
@@ -446,46 +446,37 @@ struct LocalSignedTx {
 	htlc_outputs: Vec<(HTLCOutputInCommitment, Option<Signature>, Option<HTLCSource>)>,
 }
 
-#[derive(PartialEq)]
-enum InputDescriptors {
-	RevokedOfferedHTLC,
-	RevokedReceivedHTLC,
-	OfferedHTLC,
-	ReceivedHTLC,
-	RevokedOutput, // either a revoked to_local output on commitment tx, a revoked HTLC-Timeout output or a revoked HTLC-Success output
-}
-
 /// When ChannelMonitor discovers an onchain outpoint being a step of a channel and that it needs
 /// to generate a tx to push channel state forward, we cache outpoint-solving tx material to build
 /// a new bumped one in case of lenghty confirmation delay
 #[derive(Clone, PartialEq)]
-enum InputMaterial {
+pub(crate) enum InputMaterial {
 	Revoked {
 		script: Script,
```

Comment:
Not a new issue but can you add a commit renaming this to redeemscript or witness_redeemscript or whatever?

Reply:
Done